scripts/install_cli.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
this would require changing the install command to `curl | bash` instead of `curl | sh`